src/azure-cli/azure/cli/command_modules/acr/repository.py

```diff
@@ -265,7 +265,8 @@ def acr_repository_update(cmd,
                           delete_enabled=None,
                           list_enabled=None,
                           read_enabled=None,
-                          write_enabled=None):
+                          write_enabled=None,
+                          teleport_enabled=None):
```

Comment:
is there no support for create as well?

Reply:
what do you mean? create repository goes through other API